pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -41,8 +43,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	getVersionRetryInterval = 5
+	getVersionMaxRetries    = 60
+)
+
 var (
-	logger = capnslog.NewPackageLogger("github.com/rook/rook", controllerName)
+	logger           = capnslog.NewPackageLogger("github.com/rook/rook", controllerName)
+	getVersionFailed = true
```

Comment:
Why keep track of getVersionFailed? Is that for testing?

Reply:
It's to avoid spamming the logs on every reconcile loop if the detect-version job fails. I can remove this if we'd *prefer* to spam the logs on every reconcile loop.

Reply:
If it's just for the warning message, I'd think it's ok to print on every reconciliation. It should be an intermittent issue anyway and not be a verbose error.

Reply:
Alright; I'll remove this. Simplifies the code, certainly, and if we're okay with the operator seeing this for a bit that's fine. It will mean that in the case that the detect-version job fails consistently for any reason, crash-collector reconciliations will always take a full 5 minutes, but at that point the cluster likely has larger problems.